tags.json

```
@@ -25,6 +25,7 @@
   "bookmark": ["read", "clip", "marker", "tag"],
   "box": ["cube"],
   "briefcase": ["work", "bag", "baggage", "folder"],
+  "building": ["organisations"],
```

I prefer singular, because the object is singular as well.
```suggestion
  "building": ["organisation"],
```